R/chromote_session.R

```diff
@@ -426,6 +426,17 @@ ChromoteSession <- R6Class(
       private$session_id
     },
 
+    #' @description
+    #' Create a new session that connects to the same target (i.e. page)
+    #' as this session. This is useful if the session but the target still
+    #' exists.
+    respawn = function() {
+      if (!private$is_active_) {
+        stop("Can't respawn session; target has been closed.")
+      }
+      self$parent$new_session(targetId = self$targetId)
```

Comment:
I think this should also pass `private$auto_events` to the new session.

Reply:
Should it be `private$target_id` instead of `self$targetId`?

Reply:
Yes. And I actually ran the code to check it worked 😬

Reply:
`Chromote$new_session` doesn't have the `auto_events()` parameter. Do you want me to add it, or to just call `ChromoteSession$new()` directly?

Reply:
I think it would be better to call `ChromoteSession$new()`.